TestVMAgent is currently failing

Seems to be happening somewhere in cleanup, given that all subtests are passing. See test logs below:

```
=== RUN   TestVMAgent/testExternalNodeSupportBundleCollection
    vmagent_test.go:140: Stdout of the command 'kubectl apply -f sftp-deployment.yml -n testvmagent-33g84vgj': deployment.apps/sftp created
        service/sftp created
    vmagent_test.go:86: Waiting for Deployment 'testvmagent-33g84vgj/sftp' to be realized
    vmagent_test.go:100: Waiting for SupportBundleCollection 'support-bundle-collection-external-node' to be realized
    vmagent_test.go:194: Stdout of the command 'mkdir /home/foo/upload/vmbmtest0-1': 
    vmagent_test.go:201: Stdout of the command 'tar xvf /home/foo/upload/vmbmtest0-1_support-bundle-collection-external-node.tar.gz -C /home/foo/upload/vmbmtest0-1': address
        addressgroups
        agentinfo
        appliedtogroups
        flows
        iptables
        link
        logs/agent/antrea-agent.log
        memprofile
        networkpolicies
        ovsports
        route
    vmagent_test.go:208: Stdout of the command 'ls /home/foo/upload/vmbmtest0-1': address
        addressgroups
        agentinfo
        appliedtogroups
        flows
        iptables
        link
        logs
        memprofile
        networkpolicies
        ovsports
        route
    vmagent_test.go:220: Actual files after extracting SupportBundleCollection tarball vmbmtest0-1_support-bundle-collection-external-node: [address addressgroups agentinfo appliedtogroups flows iptables link logs memprofile networkpolicies ovsports route]
    vmagent_test.go:194: Stdout of the command 'mkdir /home/foo/upload/vmbmtest0-redhat-0': 
    vmagent_test.go:201: Stdout of the command 'tar xvf /home/foo/upload/vmbmtest0-redhat-0_support-bundle-collection-external-node.tar.gz -C /home/foo/upload/vmbmtest0-redhat-0': address
        addressgroups
        agentinfo
        appliedtogroups
        flows
        iptables
        link
        logs/agent/antrea-agent.log
        memprofile
        networkpolicies
        ovsports
        route
    vmagent_test.go:208: Stdout of the command 'ls /home/foo/upload/vmbmtest0-redhat-0': address
        addressgroups
        agentinfo
        appliedtogroups
        flows
        iptables
        link
        logs
        memprofile
        networkpolicies
        ovsports
        route
    vmagent_test.go:220: Actual files after extracting SupportBundleCollection tarball vmbmtest0-redhat-0_support-bundle-collection-external-node: [address addressgroups agentinfo appliedtogroups flows iptables link logs memprofile networkpolicies ovsports route]
    vmagent_test.go:194: Stdout of the command 'mkdir /home/foo/upload/vmbmtest0-win-0': 
    vmagent_test.go:201: Stdout of the command 'tar xvf /home/foo/upload/vmbmtest0-win-0_support-bundle-collection-external-node.tar.gz -C /home/foo/upload/vmbmtest0-win-0': addressgroups
        agentinfo
        appliedtogroups
        flows
        ipconfig
        logs\\ovs\\ovs-vswitchd.log
        logs\\ovs\\ovsdb-server.log
        memprofile
        network-adapters
        networkpolicies
        ovsports
        routes
    vmagent_test.go:208: Stdout of the command 'ls /home/foo/upload/vmbmtest0-win-0': addressgroups
        agentinfo
        appliedtogroups
        flows
        ipconfig
        logs\ovs\ovs-vswitchd.log
        logs\ovs\ovsdb-server.log
        memprofile
        network-adapters
        networkpolicies
        ovsports
        routes
    vmagent_test.go:220: Actual files after extracting SupportBundleCollection tarball vmbmtest0-win-0_support-bundle-collection-external-node: [addressgroups agentinfo appliedtogroups flows ipconfig logs\ovs\ovs-vswitchd.log logs\ovs\ovsdb-server.log memprofile network-adapters networkpolicies ovsports routes]
=== CONT  TestVMAgent
    vmagent_test.go:283: TestVMAgent teardown
    vmagent_test.go:294: Verifying ExternalEntity vmbmtest0-1-e8be5, expectExists false
    vmagent_test.go:294: Verifying ExternalEntity vmbmtest0-1-e8be5, expectExists false
    vmagent_test.go:294: Verifying ExternalEntity vmbmtest0-redhat-0-3e616, expectExists false
    vmagent_test.go:294: Verifying ExternalEntity vmbmtest0-win-0-b24a8, expectExists false
    vmagent_test.go:294: Verifying ExternalEntity vmbmtest0-win-0-b24a8, expectExists false
    vmagent_test.go:272: Retry, unexpected uplink interface name, expected Ethernet0 2, got Ethernet0 2~
    vmagent_test.go:272: Retry, unexpected uplink interface name, expected Ethernet0 2, got Ethernet0 2~
    vmagent_test.go:272: Retry, unexpected uplink interface name, expected Ethernet0 2, got Ethernet0 2~
    vmagent_test.go:272: Retry, unexpected uplink interface name, expected Ethernet0 2, got Ethernet0 2~
    vmagent_test.go:272: Retry, unexpected uplink interface name, expected Ethernet0 2, got Ethernet0 2~
    vmagent_test.go:272: Retry, unexpected uplink interface name, expected Ethernet0 2, got Ethernet0 2~
    vmagent_test.go:272: Retry, unexpected uplink interface name, expected Ethernet0 2, got Ethernet0 2~
    vmagent_test.go:272: Retry, unexpected uplink interface name, expected Ethernet0 2, got Ethernet0 2~
    vmagent_test.go:281: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:281
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:288
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:83
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestVMAgent
        	Messages:   	Failed to verify uplink configuration after cleanup
    fixtures.go:317: Exporting test logs to '/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/antrea-test-logs/TestVMAgent/beforeTeardown.Aug09-05-10-38'
    fixtures.go:435: Exporting logs from vmbmtest0-1
    fixtures.go:435: Exporting logs from vmbmtest0-redhat-0
    fixtures.go:435: Exporting logs from vmbmtest0-win-0
    fixtures.go:488: Deleting 'testvmagent-33g84vgj' K8s Namespace
I0809 05:10:43.370588    5913 framework.go:739] Deleting Namespace testvmagent-33g84vgj took 4.291513ms
--- FAIL: TestVMAgent (299.49s)
    --- PASS: TestVMAgent/testExternalNode (22.43s)
    --- PASS: TestVMAgent/testExternalNodeWithANP (134.12s)
        --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM (65.63s)
            --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithTCP (26.98s)
            --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithUDP (13.42s)
            --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithICMP (19.22s)
            --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithFQDN (6.01s)
        --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM (68.49s)
            --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithTCP (28.55s)
            --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithUDP (14.86s)
            --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithICMP (19.21s)
            --- PASS: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithFQDN (5.87s)
    --- PASS: TestVMAgent/testExternalNodeSupportBundleCollection (35.21s)
FAIL
FAIL	antrea.io/antrea/test/e2e	310.976s
FAIL
```

--- a/pkg/agent/config/ipsec_authentication_mode.go
+++ b/pkg/agent/config/ipsec_authentication_mode.go
@@ -38,6 +38,9 @@ func GetIPsecConfigModes() []IPsecAuthenticationMode {
 
 // String returns value in string.
 func (am IPsecAuthenticationMode) String() string {
+	if am == IPsecAuthenticationModeInvalid {
+		return "invalid"
+	}
 	return supportedIPsecAuthenticationModeStrs[am]
 }
 

--- a/pkg/agent/util/net_windows.go
+++ b/pkg/agent/util/net_windows.go
@@ -1265,13 +1265,19 @@ func getAdaptersByName(name string) ([]adapter, error) {
 // flags to return addresses in all routing compartments.
 const GAA_FLAG_INCLUDE_ALL_COMPARTMENTS = 0x00000200
 
+// GAA_FLAG_INCLUDE_ALL_INTERFACES is used in windows.GetAdapterAddresses parameter
+// flags to return addresses for all NDIS interfaces.
+const GAA_FLAG_INCLUDE_ALL_INTERFACES = 0x00000100
+
 // adapterAddresses returns a list of IpAdapterAddresses structures. The structure
 // contains an IP adapter and flattened multiple IP addresses including unicast, anycast
 // and multicast addresses.
 // This function is copied from go/src/net/interface_windows.go, with a change that flag
-// GAA_FLAG_INCLUDE_ALL_COMPARTMENTS is introduced to query interfaces in all compartments.
+// GAA_FLAG_INCLUDE_ALL_COMPARTMENTS is introduced to query interfaces in all compartments,
+// and GAA_FLAG_INCLUDE_ALL_INTERFACES is introduced to query all NDIS interfaces even they
+// are not configured with any IP addresses, e.g., uplink.
 func adapterAddresses() ([]*windows.IpAdapterAddresses, error) {
-	flags := uint32(windows.GAA_FLAG_INCLUDE_PREFIX | GAA_FLAG_INCLUDE_ALL_COMPARTMENTS)
+	flags := uint32(windows.GAA_FLAG_INCLUDE_PREFIX | GAA_FLAG_INCLUDE_ALL_COMPARTMENTS | GAA_FLAG_INCLUDE_ALL_INTERFACES)
 	var b []byte
 	l := uint32(15000) // recommended initial size
 	for {

